deployment/paiLibrary/paiService/service_management_configuration.py

```diff
@@ -16,44 +16,40 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 
-import time
-import os
-import tempfile
+def gengerate_tmp_path():
+    import time
+    import os
+    import tempfile
 
-from ...confStorage.download import download_configuration
-from ...clusterObjectModel.cluster_object_model import cluster_object_model
+    time_in_seconds = str(int(time.time()))
+    sub_directory = "tmp-service-config-{0}".format(time_in_seconds)
+    return os.path.join(tempfile.gettempdir(), sub_directory)
 
 
+def get_cluster_object_model_from_k8s(kube_config_path):
+    from ...confStorage.download import download_configuration
+    from ...clusterObjectModel.cluster_object_model import cluster_object_model
```

Comment:
Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
[https://www.python.org/dev/peps/pep-0008/#imports](https://www.python.org/dev/peps/pep-0008/#imports)

Reply:
I see. will do that.